app/models/user.rb

```diff
@@ -0,0 +1,11 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable, :confirmable,
+         :recoverable, :rememberable, :validatable
+
+  has_many :entities, dependent: :destroy
+  has_many :groups, dependent: :destroy
+
+  validates :name, presence: true
```

Comment:
- [ ] It will be better to validate all the user input (password, ...) to make sure that anyone with bad intentions cannot compromise their app.  ✍🏻